src/main/scala/com/codacy/duplication/pmd/Cpd.scala

```diff
@@ -44,7 +49,11 @@ object Cpd extends DuplicationTool {
       languages.flatMap { language =>
         val files = File(directoryPath)
           .walk()
-          .filter(file => file.isRegularFile && language.extensions.contains(file.extension.getOrElse(file.name)))
+          .filter(
+            file =>
+              file.isRegularFile &&
+                language.extensions.contains(file.extension.getOrElse(file.name)) &&
+                file.size < MaxFileSize)
```

Comment:
It should be:
```suggestion
                file.size <= MaxFileSize)
```
right?